src/sass/components/_post-box.scss

```diff
@@ -1,7 +1,17 @@
 .post-box {
   margin-bottom: 2rem;
   
-  .cover {
+  .cover1 {
+    background-image: url('../images/latest-blog/1.jpeg');
+  }
+  .cover2 {
+    background-image: url('../images/latest-blog/2.jpeg');
+  }
+  .cover3 {
+    background-image: url('../images/latest-blog/3.jpeg');
+  }
+  
+  [class^="cover"] {
```

Comment:
Moim zdaniem lepszym rozwiązaniem byłoby zachowanie klas `cover` i `photo` a ich odnośniki do konkretnych `bg` zastosować jako klasa pomocniczna (druga)

Reply:
Poprawiłam zgodnie z sugestią, ale z ciekawości: dlaczego uważasz, że to lepsze rozwiązanie?